src/date-picker/date-picker.scss

```diff
@@ -4,16 +4,23 @@
 @import '../icon/index';
 @import '../menu/index';
 
+$iui-date-picker-cell-width: 40px;
+$iui-date-picker-cell-height: 36px;
+
 @mixin iui-date-picker {
   @include iui-dropdown;
   box-shadow: $iui-elevation-2;
```

Comment:
`@include iui-dropdown;` includes `box-shadow: $iui-elevation-2;` so this line can be removed.

Reply:
removed